res/database/Default/user/cujo.xml

```diff
@@ -920,70 +927,91 @@
 			<data country="D" year="1985" distribution="2" maingenre="9" subgenre="" flags="" blocks="1" price_mod="0.9" />
 			<ratings critics="30" speed="27" outcome="49" />
 			<children>
-				<programme id="cujo-serien-kastanienstrasse-01" product="2" licence_type="2" tmdb_id="0" imdb_id="" creator="8827" created_by="Cujo">
+				<programme id="cujo-serien-kastanienstrasse-01" product="2" licence_type="2" tmdb_id="0" imdb_id="" fictional="1" creator="8827" created_by="Cujo">

```

Comment:
Here we'd have to check with Marathony/Cujo. As far as I understood, at least some of the episodes are based on real episodes. Maybe add a TODO instead?

Reply:
not really, they're references to typical themes, there are hundreds of real episodes.